test/gtest/uct/ib/test_dc.cc

```diff
@@ -296,6 +303,10 @@ UCS_TEST_P(test_dc, dcs_ep_flush_pending, "DC_NUM_DCI=1") {
 
     iface = dc_iface(m_e1);
 
+    if (uct_dc_mlx5_iface_is_hybrid(iface)) {
+        UCS_TEST_SKIP_R("hybrid DCI policy does not support pending flush requests while ndci = 1");
```

Comment:
1. line seems too long
2. why it's not supported?

Reply:
its not supported because we also have a shared DCI, and the tests expect that we will run out of resources at some point, shared policies are hanging in this test as well

Reply:
you mean shared policies never run out of resources?

Reply:
Not so quickly, because shared policies are not relying on ndci (per-pool number of dcis) because they are not really using the pools, allocation methods are different

Reply:
is there another way to cause shared policy to run out of resources, like limiting QP size and/or number of DCIs/channels?

Reply:
I tried few things, but currently MAX_BB is not affecting DC as max_bb is queried from a temporary dci created in iface open, we will have to add an environment for that or try to use the one from RC to affect it.

To try that?

Reply:
i would expect UCX_DC_MLX5_TX_QUEUE_LEN to affect the DCI QP size